projects/dsp-ui/src/lib/viewer/values/date-value/date-value.component.spec.ts

```diff
@@ -116,6 +117,26 @@ class TestHostCreateValueComponent implements OnInit {
   }
 }
 
+/**
```

Comment:
Could you also add a test to `projects/dsp-ui/src/lib/viewer/values/date-value/date-input/date-input.component.spec.ts`? Maybe just check for the form to be valid.

Reply:
done in c05e0fd